config/config.go

```diff
@@ -171,15 +171,36 @@ func newIcingaClient(c *SignaliloConfig, l logr.Logger) (icinga2.Client, error)
 		}
 	}
 
-	client, err := icinga2.New(icinga2.WebClient{
-		URL:               c.IcingaConfig.URL,
-		Username:          c.IcingaConfig.User,
-		Password:          c.IcingaConfig.Password,
-		Debug:             c.IcingaConfig.Debug,
-		DisableKeepAlives: c.IcingaConfig.DisableKeepAlives,
-		TLSConfig:         tlsConfig})
-	if err != nil {
-		return nil, err
+	var client *icinga2.WebClient
+
+	var tmp []string
+
+	for _, ur := range c.IcingaConfig.URL {
+		st := strings.Split(ur, ",")
+		for _, b := range st {
+			tmp = append(tmp, b)
+		}
+	}
+
+	c.IcingaConfig.URL = tmp
```

Comment:
This is strictly speaking not necessary, as you can provide repeatable arguments in environment variables separated by newlines.

If we really want to keep this, the logic should go into `ConfigInitialize()` as we only need to do this once after Signalilo starts (side-note: `newIcingaClient()` is only called once as well, but logically, it doesn't make sense that `newIcingaClient()` modifies the configuration object).